AsyncDisplayKit/TextKit/ASTextKitRenderer.mm

```diff
@@ -265,7 +265,12 @@ - (void)drawInContext:(CGContextRef)context bounds:(CGRect)bounds;
   // If we use default options, we can use NSAttributedString for a
   // fast path.
   if (self.canUseFastPath) {
-    [_attributes.attributedString drawWithRect:shadowInsetBounds options:NSStringDrawingUsesLineFragmentOrigin | NSStringDrawingTruncatesLastVisibleLine context:self.stringDrawingContext];
+    CGRect drawingBounds = shadowInsetBounds;
+    // Add a fudge-factor to the height, to workaround a bug in iOS 7
+    if (AS_AT_LEAST_IOS8 == NO) {
+      drawingBounds.size.height += 3;
```

Comment:
I'm not sure. This amount was enough to cover all the common cases in Pinterest. I wish I had time to understand the _reason_ for the inconsistency – considering that the bounding rect we're using was given to us by the exact same attributed string – but alas I don't. =/